.github/workflows/ci.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
       - name: Check Code Coverage
         shell: bash
         run: |
-          MAX_SKIPPED=1 # at this moment msgData() in BosonVoucher is not covered
+          MAX_SKIPPED=2 # at this moment msgData() in BosonVoucher is not covered
```

Comment:
@anajuliabit What is the 2nd statement that we allow to skip?

Reply:
this was a mistake, thanks